hs_tools_resource/models.py

```diff
@@ -487,7 +487,7 @@ def _validate_tool_icon(cls, url):
     def create(cls, **kwargs):
         if 'value' in kwargs and "data_url" not in kwargs:
             url = kwargs["value"]
-            data_url = cls._validate_tool_icon(url)
+            data_url = cls._validate_tool_icon(url) if url else url
```

Comment:
Is a deleted url empty or None?

Reply:
empty.
perhaps that will create issues I didn't think of?

Reply:
Tt should be fine.  The model defaults the value to an empty string.  Just to be safe, change the else to return `""` instead of `url`.